tsdef/Room.d.ts

```diff
@@ -0,0 +1,52 @@
+import { MediaConnectionError, SignalingConnectionDisconnectedError, TwilioError } from './TwilioError';
```

Comment:
Please remove the signaling and media error definitions. Let's just stick with `TwilioError` for now:
```suggestion
import { TwilioError } from './TwilioError';
```